pandas/conftest.py

```diff
@@ -364,9 +364,9 @@ def unique_nulls_fixture(request):
 unique_nulls_fixture2 = unique_nulls_fixture
 
 
-TIMEZONES = [None, 'UTC', 'US/Eastern', 'Asia/Tokyo', 'dateutil/US/Pacific',
-             'dateutil/Asia/Singapore', tzutc(), tzlocal(), FixedOffset(300),
-             FixedOffset(0), FixedOffset(-300)]
+TIMEZONES = [None, 'UTC', 'Europe/London', 'US/Eastern', 'Asia/Tokyo',
```

Comment:
rather than adding this to ALL tests (we parameterize over this), can we just add it to a smaller subset of tests